deploy/secrets-store-csi-driver.yaml

```diff
@@ -47,6 +47,7 @@ spec:
             - "--v=3"
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--nodeid=$(KUBE_NODE_NAME)"
+            - "--volume=/etc/kubernetes/secrets-store-csi-providers"
```

Comment:
Can you pls also update this in the helm chart? https://github.com/deislabs/secrets-store-csi-driver/blob/master/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml#L51-L61

Reply:
hmm can we hold off on changes in this yaml until after we cut a new release or the deployment will fail since this flag is new and the previous release requires the env var.

Reply:
@malnick can you pls take a look at ^